packages/react-components/src/tooltip/src/TooltipTrigger.tsx

```diff
@@ -88,11 +96,11 @@ export function InnerTooltipTrigger({
     children,
     forwardedRef,
     ...rest
-}) {
+}: InnerTooltipTriggerProps) {
     const [isOpen, setIsOpen] = useControllableState(open, defaultOpen, false);
     const [triggerElement, setTriggerElement] = useState();
     const [overlayElement, setOverlayElement] = useState();
-    const [arrowElement, setArrowElement] = useState();
+    const [arrowElement, setArrowElement] = useState<HTMLElement>();
```

Comment:
Seems weird that you only need to specify `HtmlElement` for one of the 3 state values which contains an `HtmlElement`